tests/hikari/impl/test_entity_factory.py

```diff
@@ -3735,8 +3745,12 @@ def test_serialize_command_option_with_options(self, entity_factory_impl):
                     "name": "go home",
                     "required": False,
                     "choices": [{"name": "boo", "value": "hoo"}],
+                    "name_localizations": undefined.UNDEFINED,
+                    "description_localizations": undefined.UNDEFINED,
```

Comment:
This is not correct. It should never ever set it to `undefined.UNDEFINED`.

I think the other suggestion I added removes the code that does this